src/driver.js

```diff
@@ -0,0 +1,17 @@
+/** Driver file for the module, uncomment this file to test the changes
```

Comment:
Could move this to mocha unit/integ tests when we have tests later.

Reply:
Sure, this is temporary till we have tests in place.